File: sonicflow_app/ios-app/SonicFlow/SonicFlowExample.swift
import Foundation

struct SonicFlowExample: Identifiable, Equatable, Sendable {
    let id: String
    let title: String
    let subtitle: String
    let settings: SonicFlowSettings

    static let starterPack: [SonicFlowExample] = [
        SonicFlowExample(
            id: "focus-primer",
            title: "Focus Primer",
            subtitle: "5 min gamma warmup",
            settings: .standard(for: SonicFlowPreset.focus, durationMinutes: 5)
        ),
        SonicFlowExample(
            id: "flow-reset",
            title: "Flow Reset",
            subtitle: "5 min alpha reset",
            settings: .standard(for: SonicFlowPreset.flow, durationMinutes: 5)
        ),
        SonicFlowExample(
            id: "night-drift",
            title: "Night Drift",
            subtitle: "5 min delta wind-down",
            settings: .standard(for: SonicFlowPreset.sleep, durationMinutes: 5)
        )
    ]
}

File: sonicflow_app/android-app/app/src/main/java/com/sonicflow/app/ui/SonicFlowExample.kt
package com.sonicflow.app.ui

data class SonicFlowExample(
    val id: String,
    val title: String,
    val subtitle: String,
    val preset: SonicFlowPreset,
    val durationMinutes: Int
) {
    val ambientMix: Float = preset.defaultAmbientMix
    val pulseDepth: Float = preset.defaultPulseDepth

    companion object {
        val starterPack = listOf(
            SonicFlowExample(
                id = "focus-primer",
                title = "Focus Primer",
                subtitle = "5 min gamma warmup",
                preset = SonicFlowPreset.FOCUS,
                durationMinutes = 5
            ),
            SonicFlowExample(
                id = "flow-reset",
                title = "Flow Reset",
                subtitle = "5 min alpha reset",
                preset = SonicFlowPreset.FLOW,
                durationMinutes = 5
            ),
            SonicFlowExample(
                id = "night-drift",
                title = "Night Drift",
                subtitle = "5 min delta wind-down",
                preset = SonicFlowPreset.SLEEP,
                durationMinutes = 5
            )
        )
    }
}
